api/api/controllers/agents_controller.py

```diff
@@ -59,8 +58,8 @@ def delete_agents(pretty=False, wait_for_complete=False, list_agents='all', purg
 
 @exception_handler
 def get_all_agents(pretty=False, wait_for_complete=False, offset=0, limit=None, select=None, sort=None, search=None,
-                   status=None, q='', older_than=None, os_platform=None, os_version=None, os_name=None, manager=None,
-                   version=None, group=None, node_name=None, name=None, ip=None, registerIP=None):
+                   status=None, q='', older_than=None, manager=None, version=None, group=None, node_name=None,
+                   name=None, ip=None, registerip=None):
     """Get all agents
 
     Returns a list with the available agents.
```

Comment:
Update the docstring accordingly (`os_platform`, `os_version`, `os_name` does not exist in the function params anymore)

Reply:
registerIP has also changed

Reply:
Done